client/src/App.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+
+import Projects from './components/Projects';
+import Actions from './components/Actions';
+
+const App = () => {
+  const [projects, setProjects] = useState([]);
+  const [actions, setActions] = useState([]);
+
+  useEffect(() => {
+    async function fetchData() {
+      try {
+        const projects = await axios.get(
+          'https://web17-sprint-challenge-node-ex.herokuapp.com/api/projects'
+        );
+        if (projects) {
+          setProjects(projects.data);
+        } else {
+          console.log('error');
```

Comment:
hmm, this won't tell you much. if there's an actual error it will fall to the `catch` block, so maybe instead of checking the truthiness of `project` you could check `projects.length` since if it comes back but doesn't have the data you want, it'd be empty? then you could give a notice that `There are currently no projects-- click here to add one` or something